### PR TITLE
gas > input alert

### PR DESCRIPTION
**Pull request by skylarbarrera**
fixes RNBW-1178

PoW: http://cloud.skylarbarrera.com/Screen-Recording-2021-06-03-18-39-43.mp4

### CODE REVIEW

File: src/screens/ExchangeModal.js
Comment (brunobar79) [inline src/screens/ExchangeModal.js:298]: `checkGasvInput` or `checkGasInput`

File: src/screens/ExchangeModal.js
Comment (brunobar79) [inline src/screens/ExchangeModal.js:364]: same

**Review by brunobar79 — APPROVED** [review]
Approving with the condition of adding another case the swap sheet e2e before we merge this in cause it's extremely likely to break and even worse with upcoming EIP-1559 changes

File: src/screens/ExchangeModal.js
Comment (jinchung) [inline src/screens/ExchangeModal.js:363]: can get rid of lodash `get` and just do `selectedGasPrice?.txFee?.native?.value?.amount`

**Pull request merged**
